core/domain/question_domain.py

```diff
@@ -240,20 +267,38 @@ def update_question_data(self, question_data):
         """
         self.question_data = question_data
 
+    def update_question_status(self, status):
+        """Updates the question data of the question.
+
+        Args:
+            status: str. The status of the question.
+        """
+        self.status = status
+
 
 class QuestionSummary(object):
     """Domain object for Question Summary.
     """
-    def __init__(self, question_id, question_content):
+    def __init__(
+            self, question_id, creator_id, question_html_data,
+            question_model_last_updated=None, question_model_created_on=None):
         """Constructs a Question Summary domain object.
 
         Args:
             question_id: str. The ID of the question.
-            question_content: str. The static HTML of the question shown to
+            creator_id: str. The user ID of the creator of the question.
+            question_model_last_updated: datetime.datetime. Date and time
+                when the question model was last updated.
+            question_model_created_on: datetime.datetime. Date and time when
+                the question model is created.
+            question_html_data: str. The static HTML of the question shown to
```

Comment:
Is this the "content" field of the question's State object?

If so, should we call it question_content instead? I wasn't quite sure what this field actually referred to, especially since it seems like there might be two different meanings for "question" here (the Question object, and the question that is asked of the learner).

Reply:
I've renamed it to question_content. It is the HTML part of the content of each state (question's only state)